example/cabal.project

```diff
@@ -1,59 +1,61 @@
+allow-newer: template-haskell
+
 packages:
     ./
 
 source-repository-package
     type: git
     location: https://github.com/input-output-hk/plutus
-    subdir: language-plutus-core
-    tag: 73234550dc2b2c6209975789845bb5ef6abdf6bc
+    subdir: plutus-core
```

Comment:
you can actually have more than one `subdir` per `source-repository-package`

Reply:
Oh, that's nice! I've updated it now so they're all in a list for the plutus ones.